contributors/devel/sig-testing/monitoring.md

```diff
@@ -0,0 +1,264 @@
+
+The Kubernetes community has its own instance of TestGrid, https://testgrid.k8s.io/
+which we use to monitor and observe the health of the project.
+
+Each [SIG](https://github.com/kubernetes/community/blob/master/sig-list.md) has
+its own set of dashboard. Each dashboard is composed of different jobs (build,
+unit test, integration test, end-to-end (e2e) test, etc.) These views allow
+different teams to monitor and understand how their areas are doing.
+
+E2E jobs are in turn made up of test stages (e.g., bootstrapping a Kubernetes
+cluster, tearing down a Kubernetes cluster) and e2e tests are organized
+hierarchically per Component and Subcategory within that component. (e.g., the
+[Kubectl
+client](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/kubectl/kubectl.go#L229)
+component has tests that describe the expected behavior of [Kubectl
+logs](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/kubectl/kubectl.go#L1389)
+One such test is [should be able to retrieve and filter
+logs](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/kubectl/kubectl.go#L1412)).
+
+We highly encourage SIGs to periodically monitor the dashboards related to the
+sub-projects that they own. If you see that a job or test has been failing,
+please raise an issue with the corresponding SIG in either their mailing list or
+in Slack.
+
+Help maintaining tests is always needed throughout the project:
+the following contributions are always welcome
+ - [Triage Flaky Tests](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+kind%2Fflake)
+ - [Fix Flaky Test](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+kind%2Fflake+-label%3Aneeds-triage+)
+ 
+**Note**: It is important that all SIGs periodically monitor their jobs and
+tests.
+
+Furthermore, if jobs or tests are failing or flaking, then pull requests will
+take a lot longer to be merged.
+
+### What dashboards should I monitor?
+
+This depends on what areas of Kubernetes you want to contribute to. You should
+monitor the dashboards owned by the SIG you are working with.
+
+Additionally, you should check:
+
+* https://testgrid.k8s.io/sig-release-master-blocking 
+* https://testgrid.k8s.io/sig-release-master-informing
+
+since these jobs run tests that are used by SIG Release to determine the overall
+quality of Kubernetes and whether or not the commit on master can be considered
+suitable for release. Failing tests on a job in sig-release-master-blocking
+block a release from taking place.
+
+Also, make sure to periodically check on the "blocking" and "informing"
+dashboards for past releases. See https://testgrid.k8s.io/sig-release
+
+---
+
+## What do I do when I see a TestGrid alert?
+
+If you are part of a SIG's mailing list, occasionally you may see emails from
+TestGrid reporting that a job or a test has recently failed. If you are browsing
+through TestGrid, you may also see jobs labeled as "flaky" (in purple) or as
```

Comment:
It might cover it? 

I'm not a fan of the wording overall but I won't block on it.  For instance, I'm not sure "I happened to be browsing through" is relevant to a heading of "I see a TestGrid alert".  Maybe a separate section that describes browsing and when/how to act?